py/client-ticking/build.gradle

```
@@ -134,66 +105,60 @@ def testPyClientTickingManyLinux = { wheelsSet, taskName, parentContainer, image
-                   '''set -eux ; \
-                      for spec in ${WHEELS_SET}; do \
-                        pyver=`echo "$spec" | cut -d: -f 1`; \
-                        tag=`echo "$spec" | cut -d: -f 2`; \
-                        [ -f /project/pyt-wheels/pydeephaven_ticking*"${tag}"*.whl ]; \
-                        source "/project/$pyver/bin/activate"; \
-                        pip install unittest-xml-reporting; \
-                        pip install /project/dep-wheels/*.whl; \
-                        pip install /project/pyt-wheels/pydeephaven_ticking*"${tag}"*.whl; \
-                        deactivate; \
-                     done
-                   ''')
-
         //
         // Setup for test run.
         //
         environmentVariable 'DH_HOST', deephavenDocker.containerName.get()
         environmentVariable 'DH_PORT', '10000'
+        environmentVariable 'PREFIX', prefix
+        environmentVariable 'WHEELS_SET', wheelsSet
+        environmentVariable 'DEEPHAVEN_VERSION', project.version.toString()
+        entrypoint = ['/entrypoint.sh']
     }
     containerDependencies.dependsOn = [deephavenDocker.healthyTask]
     containerDependencies.finalizedBy = deephavenDocker.endTask
     network = deephavenDocker.networkName.get()
-    parentContainers = [ project.tasks.getByName(parentContainer),
-                         Docker.registryTask(project, "${image}") ]
-    entrypoint = ['bash', '-c',
-                  "WHEELS_SET='${wheelsSet}'; " +
-                  '''
-                  for spec in ${WHEELS_SET}; do \
-                    pyver=`echo "$spec" | cut -d: -f 1`; \
-                    source "/project/$pyver/bin/activate"; \
-                    "python$pyver" -m xmlrunner discover tests -v -o "/out/report/$pyver"; \
-                    deactivate
-                  done
-                  ''']
+    parentContainers = [ parentContainer.get(), Docker.registryTask(project, image.toString()) ]
     copyOut {
-        into layout.buildDirectory.dir('test-results')
+        into layout.buildDirectory.dir("${taskName}-test-results")
     }
 }}
 
-def testPyClientTicking = testPyClientTickingManyLinux(
-    checksWheelSet,
-    'testPyClientTicking',
-    'pyClientTicking',
-    'ubi-minimal')
+def assembleWheelsSet = [ '3.8:cp38-cp38', '3.9:cp39-cp39', '3.10:cp310-cp310', '3.11:cp311-cp311', '3.12:cp312-cp312' ]
+
+def pyClientTickingWheels = buildPyClientTickingManyLinux(
+        String.join(' ', assembleWheelsSet),
+        'pyClientTickingWheels')
+
+def testPyClientTickingUbiMinimal = testPyClientTickingManyLinux(
+        // image does not have python3.10, python3.12 package
+        String.join(' ', assembleWheelsSet - [ '3.10:cp310-cp310', '3.12:cp312-cp312' ]),
+        'testPyClientTickingUbiMinimal',
+        pyClientTickingWheels,
+        'ubi-minimal')
+
+def testPyClientTickingFedora = testPyClientTickingManyLinux(
+        String.join(' ', assembleWheelsSet),
```

note that groovy has String.join, this could be
```suggestion
        assembleWheelsSet.join(' '),
```